src/app/(admin)/dashboard/subscribers/page.tsx

```diff
@@ -0,0 +1,45 @@
+import { auth } from '@clerk/nextjs'
+import { redirect } from 'next/navigation'
+import { isAdminUser } from '@/utils/auth'
+import Dashboard from '@/components/dashboard/Dashboard'
+import { getAllDeals, getAllSubscribers } from '@/lib/queries'
+import SubscriberList from '@/components/subscriber/SubscriberList'
+
+export default async function Home() {
+  const { userId } = auth()
```

Comment:
Since the authentication check is going to be the same for each one of our dashboard pages, we should move the auth check to a `layout.tsx` file inside of the `dashboard` directory.

Reply:
Will be rolled into next push